public/css/skeleton.css

```diff
@@ -587,6 +619,14 @@ hr {
   border-top: 1px solid #e1e1e1;
 }
 
+.bad {
```

Comment:
Can we have a more meaningful name here please? How about `warning` or `danger` or something ?

Reply:
Sure, good point.